projects/pgai/pgai/semantic_catalog/gen_sql.py

```diff
@@ -270,12 +276,14 @@ def record_sql_statement(  # pyright: ignore [reportUnusedFunction]
                     error=error,
                 )
             else:
+                logger.info("the sql statement is valid")
```

Comment:
Won't this be printed as part of `validate_sql_statement` as well?

Reply:
yep. i'll remove